backend/src/Controllers/getBallotsByElectionIDController.ts

```diff
@@ -0,0 +1,30 @@
+import ServiceLocator from "../ServiceLocator";
+import Logger from "../Services/Logging/Logger";
+import { BadRequest } from "@curveball/http-errors";
+import { Ballot } from '../../../domain_model/Ballot';
+import { Score } from '../../../domain_model/Score';
+import { expectPermission } from "./controllerUtils";
+import { permissions } from '../../../domain_model/permissions';
+
+const BallotModel = ServiceLocator.ballotsDb();
+
+const getBallotsByElectionID = async (req: any, res: any, next: any) => {
+    var electionId = req.election.election_id;
+    Logger.debug(req, "getBallotsByElectionID: " + electionId);
+
+    expectPermission(req.user_auth.roles, permissions.canViewBallots)
+
+    const ballots = await BallotModel.getBallotsByElectionID(String(electionId), req);
```

Comment:
can we clarify when / why this would be null vs an empty array?
requesting ballots, and no one has voted, seems a legitimate case to handle and I wouldn't necessarily call that an error.

Reply:
Good question, currently the ballot model will only output empty array. Maybe we don't need to consider a null output.